### ISSUE TITLE
axon/examples/basics/multi_input_examples.exs failing to run

### ISSUE THREAD

**Issue by jwthomp** [elixir-nx/axon#236]
When running `elixir multi_input_examples.exs` I am getting the following error:

```
Unchecked dependencies for environment dev:
* nx (https://github.com/elixir-nx/nx.git)
  the dependency does not match the requirement "~> 0.1.0-dev", got "0.2.0"
** (Mix.Error) Can't continue due to errors on dependencies
    (mix 1.13.3) lib/mix.ex:515: Mix.raise/2
    (mix 1.13.3) lib/mix/tasks/deps.loadpaths.ex:34: Mix.Tasks.Deps.Loadpaths.run/1
    (mix 1.13.3) lib/mix/task.ex:397: anonymous fn/3 in Mix.Task.run_task/3
    (mix 1.13.3) lib/mix.ex:724: anonymous fn/2 in Mix.install/2
    (elixir 1.13.3) lib/file.ex:1560: File.cd!/2
    (mix 1.13.3) lib/mix.ex:719: Mix.install/2
    xor.exs:1: (file)
  ```

I looked at the nx repo, and none of the branches appear to support a nx version of 0.1.0-dev anymore. 

What I'm running:

```
OS: macOS 11.6.4 20G417 x86_64 (Big Sur)
Host: MacBookPro16,1
Kernel: 20.6.0
```

```
❯ elixir --version
\Erlang/OTP 24 [erts-12.1.5] [source] [64-bit] [smp:16:16] [ds:16:16:10] [async-threads:1] [jit]

Elixir 1.13.3 (compiled with Erlang/OTP 24)

❯ livebook --version
Erlang/OTP 24 [erts-12.1.5] [source] [64-bit] [smp:16:16] [ds:16:16:10] [async-threads:1] [jit]

Elixir 1.13.3 (compiled with Erlang/OTP 24)

Livebook 0.5.2
```

**Comment by josevalim** [comment]
We just released Nx v0.2.0, so things are being updated. We will bump Axon to Nx v0.2 soon and everything should get much simpler!

**Comment by jwthomp** [comment]
Thank you Jose, I cloned Axon locally and updated it so it depends on Nx v 0.2.0 and I am up and running.

Thank you to all of you who have put so much into Elixir and its ecosystem!

**Comment by seanmor5** [comment]
Closed in #250
